docs/rules/linebreak-style.md

```diff
@@ -8,46 +8,54 @@ different line endings are written by either of the mentioned (might especially
 The linebreaks (new lines) used in windows operating system are usually _carriage returns_ (CR) followed by a _line feed_ (LF) making it a _carriage return line feed_ (CRLF)
 whereas Linux and Unix use a simple _line feed_ (LF). The corresponding _control sequences_ are `"\n"` (for LF) and `"\r\n"` for (CRLF).
 
-Many versioning systems (like git and subversion) can automatically ensure the correct ending. However to cover all contingencies you can activate this rule.
+Many versioning systems (like git and subversion) can automatically ensure the correct ending. However to cover all contingencies, you can activate this rule.
 
 ## Rule Details
 
-This rule aims to ensure having consistent line endings independent of operating system, VCS or editor used across your codebase.
+This rule enforces consistent line endings independent of operating system, VCS, or editor used across your codebase.
 
-The following patterns are considered problems:
+### Options
 
-```js
-/*eslint linebreak-style: "error"*/
+This rule has a string option:
 
-var a = 'a'; // \r\n
-```
+* `"unix"` (default) enforces the usage of Unix line endings (i.e. `\n` for 'LF').
```

Comment:
Let’s write for example, … instead of e.g. … and that is, … instead of i.e. …

But in this case, maybe we can just simplify?
… Unix line endings: `\n` for LF